README.md

```diff
@@ -123,49 +123,62 @@ import CookieConsent, { Cookies, getCookieConsentValue } from "react-cookie-cons
+| contentClasses           |                   string                    | ""                                   | CSS classes to apply to the content                                                                                                     |
+| overlayClasses           |                   string                    | ""                                   | CSS classes to apply to the surrounding overlay                                                                                         |
+| style                    |                   object                    | [look at source][style]              | React styling object for the bar.                                                                                                       |
+| buttonStyle              |                   object                    | [look at source][buttonstyle]        | React styling object for the button.                                                                                                    |
+| declineButtonStyle       |                   object                    | [look at source][declinebuttonstyle] | React styling object for the decline button.                                                                                            |
+| contentStyle             |                   object                    | [look at source][contentstyle]       | React styling object for the content.                                                                                                   |
+| overlayStyle             |                   object                    | [look at source][overlaystyle]       | React styling object for the overlay.                                                                                                   |
+| disableButtonStyles      |                   boolean                   | false                                | If enabled the button will have no default style. (you can still supply style through props)                                            |
+| enableDeclineButton      |                   boolean                   | false                                | If enabled the decline button will be rendered                                                                                          |
+| flipButtons              |                   boolean                   | false                                | If enabled the accept and decline buttons will be flipped                                                                               |
+| ButtonComponent          |               React component               | button                               | React Component to render as a button.                                                                                                  |
```

Comment:
The code looks good but this change is a bit overkill imo :)

Reply:
yep, I tried to follow the same structure but as the second column is long, edit all table was necessary :)